src/AlgebraicGeometry/ToricVarieties/ToricLineBundles/cech-cohomology.jl

```diff
@@ -0,0 +1,101 @@
+#The following function is called, when algorithm="chambers" is specified for all_cohomologies
+function _all_cohomologies_via_cech(tl::ToricLineBundle)
+  our_maps = transpose.(matrix.(_toric_cech_complex(tl)))
```

Comment:
`matrix` probably produces a dense matrix, right? For the moment, the call to `rank` below probably requires dense matrices as input. But eventually, we should probably aim for use of sparse matrices here, right? 

If we returned an honest `AbsHyperComplex` of dimension `1` from `_toric_cech_complex`, then all of the computations in this routine can probably be subsumed under `simplify` and reading off the ranks of the result of that. My guess is that this will perform better for bigger examples, as it uses exclusively sparse matrices in its internals.